node-js-basics-101/node_modules/ws/lib/ErrorCodes.js

```diff
@@ -0,0 +1,28 @@
+/*!
+ * ws: a node.js websocket client
+ * Copyright(c) 2011 Einar Otto Stangvik <einaros@gmail.com>
+ * MIT Licensed
+ */
+
+'use strict';
+
+module.exports = {
```

Comment:
__[jshint]__
* 'module' is not defined.

<!-- Comment by [SideCI](https://sideci.com) -->